js/common.js

```diff
@@ -121,13 +123,13 @@ function showBuildInfo(buildRevision) {
             "limit":1,
             "format":"list",
             "select":["build.revision","build.created_timestamp","build.taskId"],
-            "from":"coverage",
+            "from":"coverage-summary",
             "where":{"eq":{"build.revision":buildRevision}}
         }));
 
         var taskId = result.data[0].build.taskId;
-        var url = `https://tools.taskcluster.net/task-inspector/#${taskId}/`;
```

Comment:
I do not what version of Javascript allows these back-quotes, but nothing I have will show this code as correct.  I wish Javascript would stop adding syntactic sugar, and solve the lack-of-threading problem.